docs/tutorials/vagrant.rst

```diff
@@ -0,0 +1,740 @@
+        vagrant ssh dev
+
+    And then within that virtual machine::
+
+        cd ~/src
+        ./scripts/clean_all.bash
+        ./scripts/reset_submodules.bash
+        ./scripts/link_install_locally.bash
+
+    .. WARNING::
+        This downloads the latest code and reinstalls it all from source, which can take a few minutes.
+
+    Once that is complete, you can exit your SSH sessions tith the VM and start all the services::
+
+        vagrant ssh dev -c "sudo nf start"
+
+When RackHD is operational, the self-hosted API documentation should immediately
+be available:
+
+- 1.1 API documentation at http://localhost:9090/docs
+- 2.0 and Redfish API documentation at http://localhost:9090/swagger-ui
+- Included task documentation at http://localhost:9090/taskdoc
+- a developer UI to live updates of RackHD at http://localhost:9090/ui
+
+The self-hosted documentation for 2.0 and the Redfish API includes the ability to
+invoke sample commands directly on your local instance through the documentation pages.
+
+You can also interact with RackHD using ``curl`` from the commandline, or a tool
+such as `Postman`_ in the browser.
+
+.. _Postman: https://www.getpostman.com
+
+With a brand new instance, you should be able to access the ``nodes/`` API endpoint
+and see an empty list of nodes.
+
+- ``curl http://localhost/api/2.0/nodes | python -m json.tool``
+
+.. code-block:: JSON
+
+    []
+
+You can also view a list of all the built-in workflows
+
+- ``curl http://localhost/api/2.0/workflows/graphs | python -m json.tool``
+
+.. code-block:: JSON
+
+    [
+        {
+            "friendlyName": "Arista Switch ZTP Discovery",
+            "injectableName": "Graph.Switch.Discovery.Arista.Ztp",
+            "tasks": [
+                {
+                    "label": "catalog-switch",
+                    "taskDefinition": {
+                        "friendlyName": "Catalog Arista Switch",
+                        "implementsTask": "Task.Base.Linux.Commands",
+                        "injectableName": "Task.Inline.Catalog.Switch.Arista",
+                        "options": {
+                            "commands": [
+                                {
+                                    "catalog": {
+                                        "format": "json",
+                                        "source": "version"
+                                    },
+                                    "downloadUrl": "/api/1.1/templates/arista-catalog-version.py"
+                                }
+                            ]
+                        },
+                        "properties": {}
+                    }
+                }
+            ]
+        },
+        ...
+
+.. sidebar:: jq
+
+    You can get details on how to use `jq`, a handy tool for parsing structured JSON data at https://stedolan.github.io/jq/manual/
+
+If you want to just see the names of the workflows:
+
+- curl http://localhost:9090/api/2.0/workflows/graphs | jq '.[]["injectableName"]'
```

Comment:
final note: wrapping the command to be invoked by the CLI with `` did the trick - set preformatting - was missing on this example